README.md

```diff
@@ -19,7 +19,9 @@ Any contributions are welcome.
 
 ### Building
 
-You can either run the whole build including linting and coding standards using
+Initially you need to run `composer install`, or `composer update` in case you are working in a folder which was built before.
```

Comment:
Should be: `in case you aren't working`

Reply:
hmm I dont think the proposal is correct.

One would have to use `composer update` when working in a folder which was built e.g. a few weeks before. otherwise you have a fresh clone and therefore you just use `composer install`.

maybe I am missing something, or misread your comment?

Reply:
The only thing that's not obvious is running `composer install` when you freshly clone the repo. Everything else is communicated. If you have old `composer.lock` when compared to updated `composer.json`, it's told to you by `composer-validate` step when running Phing (https://github.com/phpstan/phpstan-src/blob/master/build.xml#L18).

Reply:
adjusted per feedback